atlp-rwanda/e-comm-team-techsmith-bn
# #184752076 Setup Unit Testing

## What does this PR do?
- It set unit tests for continuous integration
## Description of Task to be completed?
To ensure the back end has all functionality working as intended. You may use `Mocha` as the test runner and `chai` and `chai-http `for the unit tests.
## How should this be manually tested?
- Clone this repo
- Run `npm install`
- Create a `.env` file in the root of the app and add all necessary environment variables.
   Check `.env.example` to find required environment variables need to run the application.
- Run`npm run test`
## Any background context you want to provide? 
Units tests will be run on [Circleci](https://app.circleci.com/pipelines/github/atlp-rwanda/e-comm-team-techsmith-bn?branch=ch-integrate_Circleci-184752080)  
## What are the relevant pivotal tracker stories?
- [#184752076](https://www.pivotaltracker.com/story/show/184752076)
